python/llm/example/NPU/HF-Transformers-AutoModels/LLM/CPP_Examples/README.md

```diff
@@ -91,3 +91,10 @@ AI是"人工智能"的缩写，是英文"Artificial Intelligence"的翻译。它
 
 Decode 46 tokens cost xxxx ms (avg xx.xx ms each token).
 ```
+
+### Troubleshooting
+
+#### Program crash with Chinese prompt
+If you run CPP examples on Windows and find that your program aborting or outputs abnormally when accepting Chinese prompts, you can open `Region->Administrative->Change System locale..`, check `Beta: Use Unicode UTF-8 for worldwide language support` option and then restart your computer.
+
+For detailed instructions on how to do this, see [this issue](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105600469).
```

Comment:
[this issue](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105600469). 
->
[this issue](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105598660).

Reply:
Please update the internal link.